mne/io/ctf/ctf.py

```diff
@@ -165,6 +175,16 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
                 _mult_cal_one(data_view, this_data, idx, cals, mult)
                 offset += n_read
 
+    def _clean_names(self):
+        """Clean up CTF suffixes from channel names."""
+        mapping = dict(zip(self.ch_names, _clean_names(self.ch_names)))
+
+        super(RawCTF, self).rename_channels(mapping)
```

Comment:
this is overkill (`RawCTF` does not have its own `rename_channels`), you can just do `self.rename_channels(mapping)`